Pear/Controller/HomeViewController.swift

```diff
@@ -101,6 +101,10 @@ class HomeViewController: UIViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
 
+    /*
+
+     TODO: Comment this back when feedback route is done
+    
     private func showInAppFeedback() {
```

Comment:
will give more details on this when I get updated with the current status of this route